src/genericComponents/RadioButton.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { InputGroup } from 'react-bootstrap';
+import styled from 'styled-components';
+
+const RadioButton = (props: any) => {
+  const { onClick } = props;
+
+  return <CAGRadioButton onClick={onClick}></CAGRadioButton>;
+};
+
+const CAGRadioButton = styled(InputGroup.Radio)`
+  border: 1.5px solid var(---4d5055);
```

Comment:
`var` doesn't work in styled components. Use a JS variable from our theme. Same for border color. No need for opacity. Etc.